### DISCUSSION TITLE
whisper-demo-for-web（React + FastAPI）

### DISCUSSION THREAD

**Discussion by Mumujianguang** [openai/whisper#2064] [Show and tell]
Implement front-end audio collection based on WebRTC, and call Whisper for speech recognition through the interface provided by Python

repo: https://github.com/Mumujianguang/whisper-demo-for-web

https://github.com/openai/whisper/assets/52249626/614fac39-e40f-4072-a610-4f0829db86bf

**Comment by chemistwang** [comment]
请问下本地电脑是啥配置，我的 M1 平均需要7s 
<img width="404" alt="image" src="https://github.com/openai/whisper/assets/32005560/01393522-edfc-4f1f-85b1-ab568fa17ff1">

**Reply by Mumujianguang** [reply]
录视频这台是 i5 13490F + 4060，也不稳定，偶尔2-3s
